manifests/arcos.yaml

```diff
@@ -11,24 +11,19 @@ steps:
   - manifest: rack.yaml
 
   - model: ../Boeing-Ontology/OwlModels/import.yaml
-  - nodegroups: ../nodegroups/ingestion/arcos.AH-64D
 
   - model: ../GrammaTech-Ontology/OwlModels/import.yaml
-  - nodegroups: ../nodegroups/ingestion/arcos.acert
 
   - model: ../LM-Ontology/OwlModels/import.yaml
-  - nodegroups: ../nodegroups/ingestion/arcos.certgate
 
   - model: ../SRI-Ontology/OwlModels/import.yaml
-  - nodegroups: ../nodegroups/ingestion/arcos.descert
 
   - model: ../STR-Ontology/OwlModels/import.yaml
-  - nodegroups: ../nodegroups/ingestion/arcos.arbiter
 
   - model: ../RTX-Ontology/OwlModels/import.yaml
 
   - data: ../RACK-Ontology/ontology/NIST-800-53/import.yaml
 
-  - model: ../RACK-Ontology/OwlModels/MITRE-CWE.yaml
+  - model: ../RACK-Ontology//OwlModels/MITRE-CWE.yaml
```

Comment:
This looks unnecessary.

Reply:
yikes. typo.

Reply:
I just wanted to make it lighter weight, so we don't have to load all these nodegroups into RACK-in-a-Box if we are not relying on them for ingestion.